nautobot_golden_config/tests/test_graphql.py

```diff
@@ -118,26 +118,30 @@ def setUp(self):
             git_obj = GitRepository.objects.create(**item)
             git_obj.save()
 
-        backup_repo_list = GitRepository.objects.filter(
-            provided_contents__contains="nautobot_golden_config.backupconfigs"
-        )
-        intended_repo_list = GitRepository.objects.filter(
-            provided_contents__contains="nautobot_golden_config.intendedconfigs"
-        )
-
-        GoldenConfigSetting.objects.update(
+        # Since we enforce a singleton pattern on this model, nuke the auto-created object.
```

Comment:
are we testing the auto-created "default" object in some test?

Reply:
Not sure if this is needed for the following reasons :
- successful migration means there is always a default object,
- technically there could be no plugin's settings object at all (once default is deleted)